brain_games/game.py

```diff
@@ -0,0 +1,21 @@
+from prompt import string
+
+
+def game(name, func):
+    count = 0
+    while count < 3:
+        task_params = func()
+        print(task_params["task"])
+        print(f"Question: {task_params['expression']}")
```

Comment:
these prints are repeated in both games, let's make a function for them in `cli` module

Reply:
should be replaced with functions from `cli`